beta-src/src/utils/draw-arrow.ts

```diff
@@ -0,0 +1,46 @@
+export default function drawArrow(actionType, source, svgElement, target) {
+  const actionTypeColors = {
+    moveOrder: "#FFFFFF",
+    move: "#000000",
+    moveConvoy: "#2042B8",
+    moveFailed: "#BB0000",
+    moveSupport: "#F8F83D",
+    holdSupport: "#3FC621",
+    retreat: "#BD2894",
+  };
+  const sourceNodeData = svgElement
+    .select(`#${source}`)
+    .node()
+    .getBoundingClientRect();
+  const targetNodeData = svgElement
+    .select(`#${target}`)
+    .node()
+    .getBoundingClientRect();
+
+  svgElement.select("#arrowline").remove();
+  svgElement.select("#arrowhead").remove();
```

Comment:
why are we removing these? i don't think i understand what is going on here.

Reply:
These being removed allows for the arrow to be redrawn according to the right scaling. I initially thought only one arrow can appear at a time so I did not account for multiple arrows